conf/application.conf

```diff
@@ -85,6 +85,12 @@ controllers {
     needsAuditing = false
   }
 
+  uk.gov.hmrc.individualsbenefitsandcreditsapi.controllers.controllers.SandboxMicroserviceHelloWorldController = {
```

Comment:
Is controllers.controllers correct?

Reply:
It is not, some other things i've seen to correct as well.

Reply:
@parkeradam - I think this could well be the issue in QA. Sorry I did not spot this :-)